### PR TITLE
Fix discommissioned upload artifact v2 action

### PR DESCRIPTION
**Pull request by Abacn**

### CODE REVIEW

**Comment by Abacn** [comment]
R: @damccorm

File: .github/workflows/scorecards-analysis.yml
Comment (Polber) [inline .github/workflows/scorecards-analysis.yml:1]: Could you set this one too?
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/.github/actions/publish-site-report/action.yml#L35

Not sure why it is out of sync, but probably smart to sync all the versions

**Review by Polber — APPROVED** [review]

**Comment by Abacn** [comment]
@Polber PTAL thanks!

**Review by damccorm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**